src/main.rs

```diff
@@ -463,7 +467,7 @@ fn tag_impl(repo: &Repository, matches: &clap::ArgMatches) -> i32 {
     let issue = try_or_1!(Oid::from_str(matches.value_of("issue-hash").unwrap()));
 
     // get the head for the issue to tag
-    let mut issue_head = try_or_1!(repo.get_local_issue_head(issue));
+    let mut issue_head = try_or_1!(repo.find_issue(issue).and_then(|issue| issue.find_local_head()));
```

Comment:
Same points as above (the first one only if you care, because everything fits in one line, still).